### PR TITLE
Fix linux kw

### PR DESCRIPTION
**Pull request by dseah-intel**

### CODE REVIEW

File: Source/App/EbAppConfig.c [hunk 1]
```diff
@@ -715,6 +715,11 @@ static EB_ERRORTYPE VerifySettings(EbConfig_t *config, uint32_t channelNumber)
 		return_error = EB_ErrorBadParameter;
 	}
 
+    if (config->framesToBeEncoded >= 0x7FFFFFFFFFFFFFFF) {
```
Comment (tianjunwork) [inline Source/App/EbAppConfig.c]: use LLONG_MAX in limits.h?

Comment (dseah-intel) [reply Source/App/EbAppConfig.c]: fixed :)

File: Source/Lib/Codec/EbInitialRateControlProcess.c [hunk 2]
```diff
@@ -342,15 +342,17 @@ void DetectGlobalMotion(
...
-		pictureControlSetPtr->isTilt = EB_TRUE;
-	}
+    if (totalCheckedLcus) {
+        // If more than PAN_LCU_PERCENTAGE % of LCUs are PAN
+        if ((totalPanLcus * 100 / totalCheckedLcus) > PAN_LCU_PERCENTAGE) {
+            pictureControlSetPtr->isPan = EB_TRUE;
+        }
+    }
+    if (totalCheckedLcus) {
+        if ((totalTiltLcus * 100 / totalCheckedLcus) > PAN_LCU_PERCENTAGE) {
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbInitialRateControlProcess.c]: combine two if (totalCheckedLcus > 0)?

File: Source/Lib/Codec/EbModeDecisionConfigurationProcess.c [hunk 3]
```diff
@@ -2014,7 +2015,8 @@ void DeriveLcuScore(
                         validCu8x8Count++;
                     }
                 }
-                distortion = CLIP3(pictureControlSetPtr->ParentPcsPtr->interComplexityMinPre, pictureControlSetPtr->ParentPcsPtr->interComplexityMaxPre, (distortion / validCu8x8Count) * 64);
+                if (validCu8x8Count)
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbModeDecisionConfigurationProcess.c]: if(validCu8x8Count != 0)

File: Source/Lib/Codec/EbInitialRateControlProcess.c [hunk 4]
```diff
@@ -342,15 +342,17 @@ void DetectGlobalMotion(
...
-
-	// If more than PAN_LCU_PERCENTAGE % of LCUs are PAN
-	if ((totalPanLcus * 100 / totalCheckedLcus) > PAN_LCU_PERCENTAGE) {
-		pictureControlSetPtr->isPan = EB_TRUE;
-	}
-
-	if ((totalTiltLcus * 100 / totalCheckedLcus) > PAN_LCU_PERCENTAGE) {
-		pictureControlSetPtr->isTilt = EB_TRUE;
-	}
+    if (totalCheckedLcus) {
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbInitialRateControlProcess.c]: combine two if (totalCheckedLcus)
better with if (totalCheckedLcus != 0)? And all the rest if check.

**Pull request merged**